src/MongoDB.Driver/IndexKeysDefinitionBuilder.cs

```diff
@@ -414,6 +414,30 @@ public IndexKeysDefinition<TDocument> Text(Expression<Func<TDocument, object>> f
         {
             return Text(new ExpressionFieldDefinition<TDocument>(field));
         }
+
+        /// <summary>
+        /// Creates a wildcard index key definition.
+        /// </summary>
+        /// <param name="field">The wildcard key.</param>
+        /// <returns>A wildcard index key definition.</returns>
+        public IndexKeysDefinition<TDocument> Wildcard(FieldDefinition<TDocument> field = null)
+        {
+            if (field == null)
```

Comment:
Would it make sense to move handling of null into the WildcardIndexKeyDefinition class?

That might be better encapsulation of all matters related to wildcard key definition into that single class.

Reply:
Agree. I didn't do it because originally this logic was in `DirectionalIndexKeyDefinition` and I didn't want to make `_field`  optional for all cases when this class is used.
But since we moved this logic into the new class, it's not a problem anymore. 
Done.